pkg/authorization/cache/clusterpolicy.go

```diff
@@ -24,7 +24,7 @@ type readOnlyClusterPolicyCache struct {
 	keyFunc cache.KeyFunc
 }
 
-func NewReadOnlyClusterPolicyCache(registry clusterpolicyregistry.WatchingRegistry) readOnlyClusterPolicyCache {
+func NewReadOnlyClusterPolicyCache(registry clusterpolicyregistry.WatchingRegistry) *readOnlyClusterPolicyCache {
```

Comment:
the reflector is already a pointer, so I'm surprised it matters, but if you're ok with the cache being a pointer, I don't care